### DISCUSSION TITLE
请问window.g_umi有什么办法可以禁用掉吗？还有可以隐藏掉umi version的注释吗？

### DISCUSSION THREAD

**Discussion by Sunry6** [umijs/umi#10929] [Q&A] [answered]
<img width="465" alt="6" src="https://user-images.githubusercontent.com/40784791/230844956-121e755c-8968-4088-bafa-dc42f58c9e36.png">
<img width="491" alt="66" src="https://user-images.githubusercontent.com/40784791/230845155-e64f50cf-c7cd-4d94-8b57-d0f08482039d.png">

**Comment by fz6m** [comment]
这个不会影响你使用的，如果不需要可以在构建脚本最后手动 replace 把产物 html 操作一下去掉这个。

**Reply by Sunry6** [reply]
> 这个不会影响你使用的，如果不需要可以在构建脚本最后手动 replace 把产物 html 操作一下去掉这个。

方便问一下要怎么手动把产物html里的信息去掉？

**Reply by fz6m** [reply, marked as answer]
用任意一种编程语言读一下产物的 html 文件，再 replace 或者正则替换掉。

**Reply by Sunry6** [reply]
> 用任意一种编程语言读一下产物的 html 文件，再 replace 或者正则替换掉。

好的，感谢
